common/models/update_service_request.go

```diff
@@ -1,5 +1,44 @@
 package models
 
+import (
+	"fmt"
+
+	swagger "github.com/zpatrick/go-plugin-swagger"
+)
+
 type UpdateServiceRequest struct {
-	DeployID string `json:"deploy_id"`
+	ServiceID         string  `json:"service_id"`
+	DeployID          *string `json:"deploy_id"`
+	ServiceScaleCount *int    `json:"service_scale_count"`
+}
+
+func (u UpdateServiceRequest) Validate() error {
+	if u.ServiceID == "" {
+		return fmt.Errorf("ServiceID must be specified")
+	}
+
+	if u.DeployID != nil {
```

Comment:
`if u.DeployID != nil && *u.DeployID == "" {}`

- same for `u.ServiceScaleCount`

Reply:
✅